Src/ReleaseHistory.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+- FPC: Improving RabbitMQ regex (removing new lines and spaces) from secret.
```

Comment:
>FPC [](http://example.com/codeflow?start=2&length=3)

Where's the explanatory table for this code? Can you put a copy of it at the top of the table?

Reply:
done!